src/benchmarklib/cli_config_parser.cpp

```diff
@@ -40,15 +40,8 @@ BenchmarkConfig CLIConfigParser::parse_basic_options_json_config(const nlohmann:
   if (!output_file_string.empty()) {
     output_file_path = output_file_string;
     std::cout << "- Writing benchmark results to '" << *output_file_path << "'" << std::endl;
-  } else {
-    std::cout << "- Writing benchmark results to stdout" << std::endl;
```

Comment:
I always liked it that each benchmark option printed a line at benchmark startup. What will this print if `-o` is not set? Nothing?

Reply:
Correct. The previous output was wrong - we always print the readable output to the console and never print the JSON to stdout. I don't feel like arguing about this for too long, but I rather not add lines to the output that contain no information.

Reply:
don't feel like arguing about this either. leave it as it is then